doc/builders/special/darwin-builder.section.md

```diff
@@ -61,3 +61,37 @@ builders-use-substitutes = true
 ```ShellSession
 $ sudo launchctl kickstart -k system/org.nixos.nix-daemon
 ```
+
+## As a NixOS module
+
+The `darwin.builder` bootstrapper runs a NixOS module with sensible defaults for
+two settings:
+
+- `services.macos-builder.diskSize` &mdash; The disk space used by the VM in MB,
+  with a default of 20 \* 1024 (20 GB).
+- `services.macos-builder.memorySize` &mdash; The memory used by the VM in MB,
+  with a default of 3 \* 1024 (3 GB).
+
+To provide different values for these settings, you need to use the builder as a
+NixOS VM on your own. Here's an example configuration:
```

Comment:
fyi
We could expose a function via `passthru` that allows users to add a module easily to `darwin.builder` and get back the package. I'm not convinced that that's an improvement though, because that would make darwin builder configs "special" in the experience of some users, for very little ease of use benefit. Going through general NixOS interfaces avoids that.